src/app/clusters/scenes/scenes.c

```diff
@@ -0,0 +1,1187 @@
+/**
```

Comment:
Should this dir be called `scenes-server`?

Reply:
again, just taking the names as-is from SiLabs.  think fixing could be a followup PR?

Reply:
Followup is fine by me.